modules/Bio/EnsEMBL/Analysis/Hive/Config/GenblastHomology.pm

```diff
@@ -0,0 +1,532 @@
+    genewise_path => catfile($self->o('binary_base'), 'genewise'),
+    'genblast_path'     => catfile($self->o('binary_base'), 'genblast'),
+    'genblast_eval'     => $self->o('blast_type') eq 'wu' ? '1e-20' : '1e-1',
+    'genblast_cov'      => '0.5',
+    'genblast_pid'      => '30',
+    'genblast_max_rank' => '5',
+    'genblast_flag_small_introns' => 1,
+    'genblast_flag_subpar_models' => 0,
+
+
+##################################
+# Memory settings for the analyses
+##################################
+    'default_mem'          => '900',
+    'genblast_mem'         => '1900',
+    'genblast_retry_mem'   => '4900',
+
+    
+########################
+# db info
+########################
+
+    'genblast_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_genblast_'.$self->o('release_number'),
+      -host   => $self->o('genblast_db_server'),
+      -port   => $self->o('genblast_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+
+    'genblast_nr_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_genblast_nr_'.$self->o('release_number'),
+      -host   => $self->o('genblast_db_server'),
+      -port   => $self->o('genblast_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'killlist_db' => {
+      -dbname => $self->o('killlist_db_name'),
+      -host   => $self->o('killlist_db_server'),
+      -port   => $self->o('killlist_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -driver => $self->o('hive_driver'),
+    },
+
+  };
+}
+
+sub pipeline_create_commands {
+    my ($self) = @_;
+
+    return [
+    # inheriting database and hive tables' creation
+      @{$self->SUPER::pipeline_create_commands},
+
+      $self->hive_data_table('protein', $self->o('uniprot_table_name')),
+      
+      'mkdir -p '.$self->o('genome_dumps'),
+
+    ];
+}
+
+
+sub pipeline_wide_parameters {
+  my ($self) = @_;
+
+  return {
+    %{$self->SUPER::pipeline_wide_parameters},
+    wide_ensembl_release => $self->o('ensembl_release'),
+    
+  }
+}
+
+## See diagram for pipeline structure
+sub pipeline_analyses {
+    my ($self) = @_;
+
+    my %genblast_params = (
+      wu    => '-P wublast -gff -e #blast_eval# -c #blast_cov#',
+      ncbi  => '-P blast -gff -e #blast_eval# -c #blast_cov# -W 3 -softmask -scodon 50 -i 30 -x 10 -n 30 -d 200000 -g T',
+      wu_genome    => '-P wublast -gff -e #blast_eval# -c #blast_cov#',
+      ncbi_genome  => '-P blast -gff -e #blast_eval# -c #blast_cov# -W 3 -softmask -scodon 50 -i 30 -x 10 -n 30 -d 200000 -g T',
+      wu_projection    => '-P wublast -gff -e #blast_eval# -c #blast_cov# -n 100 -x 5 ',
+      ncbi_projection  => '-P blast -gff -e #blast_eval# -c #blast_cov# -W 3 -scodon 50 -i 30 -x 10 -n 30 -d 200000 -g T',
```

Comment:
Only wu_genome and ncbi_genome should be kept, the other lines can be removed as they are not used

Reply:
Requested changes made